docs/dev/makefile.rst

```diff
@@ -0,0 +1,221 @@
+.. _makefile:
```

Comment:
this is not correct, you can build without graphviz /...

> if a third party converter is missed, the build process will spit out a message about and continues with a lower quality in the output. Even if the output is e.g. a raw SVG or DOT, in some use cases it might be a solution that fits

see [Scalable figure and image handling](https://return42.github.io/linuxdoc/linuxdoc-howto/kfigure.html)

I guess most developer or admins will never need to build with the full buildchain installed. Thats why I added the [buildhosts article](https://return42.github.io/searx/admin/buildhosts.html) article.

BTW: the math to svg conversion process is handled similar ..  it spits out warnings (see buiildhost article)  if a tool is missed, but the build process will continue (its a warning not an error).